### PR TITLE
drivers: adc: stm32: Uncrustify

### PR DESCRIPTION
**Pull request by pdietl**
Signed-off-by: Pete Dietl <petedietl@gmail.com>

### CODE REVIEW

**Comment by pdietl** [comment]
I need help from a maintainer, it seems that there is a false positive in the patch format checking.

File: drivers/adc/adc_stm32.c [hunk 1]
```diff
@@ -220,9 +220,9 @@ static const uint32_t table_samp_time[] = {
 	SMP_TIME(810, S_5),
 };
 #elif defined(CONFIG_SOC_SERIES_STM32U5X)
-static const uint16_t acq_time_tbl[8] = {5, 6, 12, 20, 36, 68, 391, 814};
+static const uint16_t acq_time_tbl[8] = { 5, 6, 12, 20, 36, 68, 391, 814 };
 static const uint32_t table_samp_time[] = {
-	SMP_TIME(5,),
+	SMP_TIME(5, ),
```
Comment (erwango) [inline drivers/adc/adc_stm32.c:225]: This is where CI is complaining. Is that strictly required ?

File: drivers/adc/adc_stm32.c [hunk 2]
```diff
@@ -1046,18 +1047,18 @@ static int adc_stm32_init(const struct device *dev)
...
-	} else   /*Case ADC3*/ {
+	} else { /*Case ADC3*/
 		channel_offset = 16UL;
 	}
 	/* Read factory calibration factors */
 	for (uint32_t count = 0UL; count < ADC_LINEAR_CALIB_REG_COUNT; count++) {
 		linear_calib_buffer = *(uint32_t *)(
 			ADC_LINEAR_CALIB_REG_1_ADDR + channel_offset + count
-		);
+			);
```
Comment (erwango) [inline drivers/adc/adc_stm32.c:1057]: While you're at, I think this break is superfluous and could be removed.
(in a dedicated commit, if you don't want to break the uncrustify only commit)

File: drivers/adc/adc_stm32.c [hunk 3]
```diff
@@ -1046,18 +1047,18 @@ static int adc_stm32_init(const struct device *dev)
...
 		LL_ADC_SetCalibrationLinearFactor(
 			adc, LL_ADC_CALIB_LINEARITY_WORD1 << count,
-			linear_calib_buffer
-		);
+				linear_calib_buffer
+			);
```
Comment (erwango) [inline drivers/adc/adc_stm32.c:1061]: ditto

**Review by erwango — CHANGES_REQUESTED** [review]

**Review by gmarull — COMMENTED** [review]
We're actually debating about dropping uncrustify in favor of clang-format, ref https://github.com/zephyrproject-rtos/zephyr/pull/41307

**Comment by pdietl** [comment]
ok I can just close this then

**Pull request closed without merging**